### PR TITLE
Added code blocks highlighting for issue #86

### PR DESCRIPTION
**Pull request by ghost**
The change is in the centralized doc.js and will dynamically load highlighting css/js, which does not require to change all html files.

It is using the https://highlightjs.org/ as tommywo suggested in original post (with github css style). Please visit https://yang458567.github.io/docs/ for sample. 

Thanks.

### CODE REVIEW

**Comment by ghost** [comment]
BTW, I had signed the Contributor License Agreement.

File: github.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/*
+
+github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```
Comment (andreastt) [inline github.css:3]: We can’t include code under an unknown license.

Comment (ghost) [reply github.css:3]: The change is using a js lib (https://github.com/isagalaev/highlight.js) suggested by original post of issue 86. It is using BSD license. 

Does the Selenium docs project require authoring group of any 3rd party open source libs to sign same contributor license agreement? I'm thinking that maybe that's why the doc.js and all html files are written from scratch.

Please kindly let me know and I will close the request for that. Thank you.

Comment (andreastt) [reply github.css:3]: Selenium doesn’t require third-party libraries to carry the same license, and a BSD license is obviously fine to include.

Can you include a reference to it in [attr.html](https://github.com/SeleniumHQ/docs/blob/gh-pages/attr.html)?

Comment (ghost) [reply github.css:3]: Thanks for the comments. Just added a "Third-Party Software" table in attr.html. I google searched such format and followed same 3-column format used by other software ([sample here](https://confluence.jetbrains.com/display/IDEADEV/Third-Party+Software+Used+by+IntelliJ+IDEA+15)).

**Comment by andreastt** [comment]
Thanks!

**Pull request merged**